test/scripts/deploy-ramen

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import os
+import tempfile
+import concurrent.futures
+from subprocess import run
+
+TOP_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+IMAGE = "quay.io/ramendr/ramen-operator:latest"
+HUB = "hub"
```

Comment:
(question) This is specific to RDR, but as the dev doc states this is the current state, and so in the future the values for the clusters would potentially come from the env file that was used?

Reply:
Why is this specific to RDR? We don't have yet the MDR version, but I think it will
have also hub, dr1 and dr2 where we install ramen. The difference is the configuration
(ramen-config/deplopy) which already supports multiple configurations.

Reply:
Let me state my question differently, would we in the future replace the cluster names with ones from a specific env on which we want to deploy? (BTW, this does not hold back the merge or anything, it is just a question on how you are thinking about this in the future)

Reply:
For deploying on a single environment we will always have the same cluster names, but
when using multiple environments we will likely have different prefixes:

    rdr-hub, rdr-dr1, rdr-dr2
    mdr-hub, mdr-dr1, mdr-dr2
    test-uuid-hub, test-uuid-dr1, test-uuid-dr2

The prefix works nicely for all cases including external clusters so I think adding
a --name-prefix to the script will be enough to handle all cases. If not we can add
arguments like:

    --hub my-hub and --clusters my-cluster1,my-cluster2

Or pass the enviroment file to the script, and add there mapping like:

    metadata:
      ramen:
         hub: my-hub
         clusters:
           - my-cluster1
           - my-cluster2

Lets work on this when the basic test is finished.

Reply:
Added https://github.com/RamenDR/ramen/issues/862